src/einsteinpy/tests/test_symbolic/test_tensor.py

```diff
@@ -126,55 +136,63 @@ def test_Tensor_repr():
     machine_representation = repr(obj1)
     assert not "object at 0x" in machine_representation
 
-
+@xfail(raises=TypeError)
 def test_TypeError2():
     scht = schwarzschild_tensor().tensor()
+    
     # pass non str object
+    
     try:
         obj = Tensor(scht, config=0)
         assert False
     except TypeError:
         assert True
 
 
+@xfail(raises=TypeError)
```

Comment:
```suggestion
@xfail(raises=TypeError, strict=True)
```
same reason mentioned above

Reply:
done